client/src/Login.tsx

```diff
@@ -49,12 +49,21 @@ const initialFormData: UserLoginData = {
     email: '',
     password: '',
 };
+
 function Login() {
     const classes = useStyles();
     const history = useHistory();
-
+    const [status, setStatus] = React.useState<'idle' | 'loading' | 'success' | 'failure'>('idle')
+    const [error, setError] = React.useState<'' | 'email' | 'password'>('')
```

Comment:
In order to differentiate between an email error and a password error, I do still need to track the type, but I have changed it so that the error message displayed is what is sent from the backend.